packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -15,6 +15,52 @@ import {
  */
 const Media = {};
 
+Media.localTracks = (audio, video, stream) => {
+  Media.localAudioTracks(audio, stream);
+  Media.localVideoTracks(video, stream);
+};
+
+Media.localAudioTracks = (audio, stream) => {
+  let audioTrack;
+  if (stream) {
+    if (stream.getAudioTracks) {
+      [audioTrack] = stream.getAudioTracks();
+    }
+  }
+  if (audioTrack) {
+    audioTrack.enabled = !audio.mute;
+    return true;
+  }
+  return false;
+};
+
+Media.localVideoTracks = (video, stream) => {
+  let videoTrack;
+  if (stream) {
+    if (stream.getVideoTracks) {
+      [videoTrack] = stream.getVideoTracks();
+    }
+  }
+  if (videoTrack) {
+    videoTrack.enabled = !video.mute;
+    return true;
+  }
+  return false;
+};
+
+Media.generateLocalMedias = (mediaId, audio, video) => {
+  if (mediaId) {
+    return [{
+      localSdp: JSON.stringify({
+        audioMuted: audio.mute,
+        videoMuted: video.mute
+      }),
+      mediaId
+    }];
+  }
+  return {};
```

Comment:
We shouldn't return two different types `[]` vs `{}`

Reply:
yep, teehee, stole this one direct from web client